cbcontainers/state/hardening/hardening_state_applier_test.go

```diff
@@ -201,7 +201,7 @@ func TestEnforcerWebhookIsApplied(t *testing.T) {
 
 func TestEnforcerWebhookIsDeleted(t *testing.T) {
 	assertValidatingWebhookIsDeleted := func(t *testing.T, appliedObjects, deletedObjects []K8sObjectDetails, webhookObject K8sObjectDetails) {
```

Comment:
This now detects that both webhooks are deleted, correct? We should rename it to just `assertWebhooksAreDeleted` then

Reply:
agree